apps/crn-server/src/data-providers/contentful/team.data-provider.ts

```diff
@@ -205,33 +271,28 @@ export const parseContentfulGraphQlTeams = (item: TeamItem): TeamDataObject => {
   const members: TeamMember[] = (
     item.linkedFrom?.teamMembershipCollection?.items || []
   ).reduce(
-    (userList: TeamMember[], membership: Membership | null): TeamMember[] => {
+    (
+      userList: TeamMember[],
+      membership: MembershipTeamById | null,
+    ): TeamMember[] => {
       if (
         !membership ||
-        !membership.linkedFrom?.usersCollection?.items[0]?.onboarded
+        !membership.linkedFrom?.usersCollection?.items[0]?.onboarded ||
+        !membership.role
       ) {
         return userList;
       }
 
       const { role, inactiveSinceDate } = membership;
       const {
-        sys,
+        sys: { id },
         firstName,
         lastName,
         email,
         avatar,
         labsCollection,
         alumniSinceDate,
       } = membership.linkedFrom?.usersCollection?.items[0] || {};
-      const id = sys?.id;
-
-      if (!id) {
-        return userList;
-      }
-
-      if (!role || !isTeamRole(role)) {
```

Comment:
because of this discussion https://yld.slack.com/archives/G01A6J33G8H/p1701088240393909

<img width="795" alt="Screenshot 2023-11-30 at 07 30 10" src="https://github.com/yldio/asap-hub/assets/16595804/cd26d0f8-bfc2-4d68-b712-b573784eb5b3">

in Contentful the team role is a dropdown so it is really impossible to select an invalid role and I thought it wasn't worth it the time to discuss if we could keep it or not 😅